src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -74,7 +74,7 @@ export class MeetingManager implements AudioVideoObserver {
 
   devicePermissionStatus = DevicePermissionStatus.UNSET;
 
-  devicePermissionsObservers: ((permission: string) => void)[] = [];
+  devicePermissionsObservers: ((permission: DevicePermissionStatus) => void)[] = [];
```

Comment:
Should we keep `string` and `DevicePermissionStatus` for now and remove `string` in v2 release? Not sure how builders may have used this as of now.

Reply:
The demo change is fine but concerned about the library hook change.

Reply:
This is considered a breaking change for v2. This is that change.

Reply:
On second thought, I'm retargeting this commit to the express branch, which is holding other breaking changes until code freeze, where we can merge them all in.

Reply:
Sounds good!